src/pages/LoginToAdminPage.js

```diff
@@ -76,12 +77,18 @@ function LoginToAdminPage() {
     }
     // UseEffect content will disply in the first place one time.
     // if some state is inside of array [] and updated, useEffect will be called again.  
-    useEffect(() => {
+    useEffect( async () => {
         const storedPassword = sessionStorage.getItem('storedPassword')
         console.log('>>>',storedPassword)
         if( storedPassword ) {
             console.log('>>>effect called one time');
-            getSuccessfullFechedData(storedPassword)
+            try {
+                await getSuccessfullFechedData(storedPassword)
+            } catch (error) {
+                console.log(error);
+                window.location = "/500"
```

Comment:
When it fails, I think we should do "nothing". Try/catch is used only for preventing the app from crashing.
So the user will see the password input form UI and put the correct password. 
If we take this user to the /500 page when this fails, the user will be stuck in the dead lock. (in a state where it's not recoverable)